app/controllers/events_controller.rb

```diff
@@ -0,0 +1,32 @@
+class EventsController < ApplicationController
+  before_action :set_event, only: [:update]
+
+  def update
+    if @event.update(event_params)
+      redirect_to tasks_path
+    else
+      redirect_to tasks_path
```

Comment:
__[rubocop]__
* Style/IdenticalConditionalBranches: Move `redirect_to tasks_path` out of the conditional.